react/features/base/participants/functions.js

```diff
@@ -45,3 +50,70 @@ function _getParticipants(participantsOrGetState) {
 
     return participants || [];
 }
+
+/**
+ * Returns the URL of the image for the avatar of a particular participant
+ * identified by their id and/or e-mail address.
+ *
+ * @param {string} [participantId] - Participant's id.
+ * @param {Object} [options] - The optional arguments.
+ * @param {string} [options.avatarId] - Participant's avatar id.
+ * @param {string} [options.avatarUrl] - Participant's avatar url.
+ * @param {string} [options.email] - Participant's email.
+ * @returns {string} The URL of the image for the avatar of the participant
+ * identified by the specified participantId and/or email.
+ *
+ * @public
+ */
+export function getAvatarURL(participantId, options = {}) {
+    // If disableThirdPartyRequests is enabled we shouldn't use third party
+    // avatar services, we are returning one of our images.
+    if (typeof config === 'object' && config.disableThirdPartyRequests) {
+        return 'images/avatar2.png';
+    }
+
+    const { avatarId, avatarUrl, email } = options;
```

Comment:
I suppose it's not introduced by you but I'd like to mention it anyway because we're dealing with these in this PR. On top of it we have it in our coding style in a way. I find it disadvantageous to use different character casings for one and the same thing. I believe It requires additional and unnecessary effort when reading and writing to pay attention to the two different character casings. I'll make sure to clean up the use of `URL` and `Url`, 'ID' and 'Id' in the react/ source code including this PR.